pkg/fftypes/jsondata.go

```diff
@@ -75,6 +75,8 @@ func (jd JSONObject) GetStringOk(key string) (string, bool) {
 		return strconv.FormatBool(vt), true
 	case float64:
 		return strconv.FormatFloat(vt, 'f', -1, 64), true
+	case nil:
+		return "", false // no need to log for nil
```

Comment:
The logs that were previously generated by not having this were always very scary looking. Should make debugging easier. Thanks!